### DISCUSSION TITLE
Ubuntu 22 client is not the latest version

### DISCUSSION THREAD

**Discussion by Nardol** [BearWare/TeamTalk5#2242] [General]
Hello,

When launching the TeamTalk client for Ubuntu 22, I have the dialog specifying version 5.16.0.5139 is available.
I click Yes, download and install the version but the dialog is showed again because in Help`->`About, the version is 5.16.0.5138.
Could it be possible to upload TeamTalk 5.16.0.5139 for Ubuntu 22 in place of the actual 5.16.0.5138? Even if actual file name looks right, it is not the right version which is available on bearware.dk.
And the main title bar still specify "unreleased".

**Comment by bear101** [comment]
hm, I don't see this issue when I download and run TeamTalk v5.16

**Reply by Nardol** [reply]
I have not tested with Mac or Windows so I don't know if it is only the case for Ubuntu, but I tested multiple times by downloading https://bearware.dk/teamtalk/v5.16/teamtalk-v5.16-ubuntu22-x86_64.tgz

**Reply by Nardol** [reply]
Works now.
I have proceeded exactly the same as my last attempt And now I have the right version

**Reply by bear101** [reply]
I also downloaded this version and it says 5.16.0.5139 in about dialog
